forms/layout_table_flexbox.css

```diff
@@ -0,0 +1,286 @@
+
+}
+.header-bottom {
+	height: 6.8vh;
+	background: #2FCBAB;
+	font-size: 1.2rem;
+}
+.header-bottom-content {
+	justify-content: left;
+}
+
+/*CSS specific for main for user table starts here*/
+main {
+	display: flex;
+	height: 75vh;
+	color: #6A6A6A;
+}
+#main-content {
+	display: flex;
+	flex-basis: 0;
+	flex-grow: 16;
+	flex-shrink: 1;
+	padding-top: 40px;
+	flex-direction: row;
+	align-items: left;
+	flex-flow: row wrap;
+	align-content: flex-start;
+}
+#main-top-content, #main-center-content {
+	display: flex;
+	flex: 1 0 100%;
+	justify-content: space-between;
+	margin-top: 10px;
+	margin-bottom: 10px;
+}
+#main-top-left {
+	flex-grow: 1;
+}
+.search-bar {
+	margin: auto;
+	background: #F3F3F3;
+}
+#main-top-right {
+	display: flex;
+	flex-grow: 2;
+	justify-content: flex-end;
+}
+.button {
+	padding: 10px;
+	font-size: 1rem;
+	white-space: nowrap;
+	text-transform: uppercase;
+	background: #2FCBAB;
+	color: white;
+	cursor: pointer;
+	box-shadow: 0 2px 4px 0 rgba(0,0,0,.5);
+}
+#main-center-content {
+	flex-flow: row wrap;
+}
+#user-table {
+	display: flex;
+	flex: 1 0 100%;
+	flex-flow: row wrap;
+	margin-bottom: 40px;
+}
+#user-table-header {
+	display: flex;
+	flex: 1 0 100%;
+	height: 6vh;
+	justify-content: space-between;
+	align-items: center;
+	background: #F3F3F3;
+	font-weight: bold;
+}
+.user-table-row {
+	display: flex;
+	flex: 1 0 100%;
+	height: 6vh;
+	justify-content: space-between;
+	align-items: center;
+	border-bottom: 1px solid #D8D8D8;
+}
+.user-table-user {
+	flex-grow: 2;
+}
+.user-table-role {
+	flex-grow: 2;
+}
+.user-table-mail {
+	flex-grow: 2;
+}
+.user-table-menu-button {
+	flex-grow: 1;
+	text-align: center;
+}
+.role {
+	display: inline-block;
+	height: 50px;
+	width: 50px;
+	line-height: 50px;
+	border-radius: 50%;
+	text-align: center;
+	font-size: 1.5rem;
+}
+.role-user {
+	background: #91D791;
+	color: white;
+}
+.role-user-inactive {
+	background: #D8F1D8;
+	color: white;
+}
+.role-user-none {
+	background: white;
+	color: white;
+}
+.role-supplier {
+	background: #8A79BA;
+	color: white;
+}
+.role-supplier-inactive {
+	background: #D6D0E7;
+	color: white;
+}
+.role-supplier-none {
+	background: white;
+	color: white;
+}
+.role-admin {
+	background: #FBE887;
+	color: white;
+}
+.role-admin-inactive {
+	background: #FEF7D5;
+	color: white;
+}
+.role-admin-none {
+	background: white;
+	color: white;
+}
+.role-manager {
+	background: #FC7878;
+	color: white;
+}
+.role-manager-inactive {
+	background: #FED0D0;
+	color: white;
+}
+.role-manager-none {
+	background: white;
+	color: white;
+}
+.role-add {
+	border: 2px dashed #A6A6A6;
+	background: white;
+	color: #A6A6A6;
+}
+.role-add-inactive {
+	border: 2px dashed #A6A6A6;
+	background: white;
+	color: white;
+}
+.role-add-none {
+	border: 2px dashed white;
+	background: white;
+	color: white;
+}
+.role-save {
+	vertical-align: top;/*moet beter kunnen, komt doordat de font-size van de save knop 1rem is ipv 1.5. Breedte van deze knop klopt ook niet.*/
```

Comment:
Alternative voor vertical-align: top;
In parent element:

display: flex;
justify-content: space-between;